mix.exs

```diff
@@ -67,7 +67,8 @@ defmodule ImgProxy.MixProject do
   defp deps do
     [
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.25", only: :dev, runtime: false}
+      {:ex_doc, "~> 0.25", only: :dev, runtime: false},
+      {:mix_test_watch, "~> 1.0", only: [:dev, :test], runtime: false}
```

Comment:
My preference would be to move any changes to dependencies to a different PR just to keep the PR's consistent in theme/purpose.